examples/dialogflow-intents-example/bot_builder.py

```diff
@@ -0,0 +1,76 @@
+# ==============================================================================
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+r"""Tool to automatically build DF bots.
```

Comment:
This example primarily focuses on automatically building Intent files based on user phrases rather than building bot.  Rephrase the module description accordingly